src/m365/base/TeamsCallReport.ts

```diff
@@ -0,0 +1,91 @@
+import { Logger } from '../../cli';
+import {
+  CommandOption
+} from '../../Command';
+import GlobalOptions from '../../GlobalOptions';
+import request from '../../request';
+import Utils from '../../Utils';
+import GraphCommand from "./GraphCommand";
+
+interface CommandArgs {
+  options: DateTimeOptions;
+}
+
+interface DateTimeOptions extends GlobalOptions {
+  fromDateTime: string;
+  toDateTime?: string;
+}
+
+export default abstract class TeamsCallReport extends GraphCommand {
```

Comment:
Aha! So, I was going to call it something like `TeamsReportPstnCallsCommand` but thought the sister issue #1772 could use the same class as a base (I was planning on taking that on after this, but its already taken :smile:). But happy to rename it if that's easier.

Reply:
I think we need to check it, because the link to the Graph API that we have in the issue points to the same API as your command and the other action is nowhere to be seen so I wonder if anything has changed that would make the other command not possible to build at the moment. I'd suggest, let's keep both command separate and then we can refactor later when we understand what's possible and how the API works. Deal?

Reply:
So there is a similar report under request #1772 that someone else is working on, but I think the output values from the command will differ slightly.

Reply:
Missed your reply above. Yep, lets keep separate.